src/SprykerMiddleware/Zed/Process/Communication/ProcessCommunicationFactory.php

```diff
@@ -153,4 +167,36 @@ protected function getRegisteredPostProcessorsList(): array
     {
         return [];
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\ProcessSettingsTransfer $processSettingsTransfer
+     *
+     * @return \Spryker\Shared\Log\Config\LoggerConfigInterface
+     */
+    protected function getProcessLoggerConfig(ProcessSettingsTransfer $processSettingsTransfer): LoggerConfigInterface
+    {
+        $configuration = $this->getProcessLoggerConfigList();
+        if (isset($configuration[$processSettingsTransfer->getName()])) {
+            return $configuration[$processSettingsTransfer->getName()]($processSettingsTransfer->getLoggerSettings());
+        }
+        return $this->createMiddlewareLogConfig($processSettingsTransfer->getLoggerSettings());
+    }
+
+    /**
+     * @return array
+     */
+    protected function getProcessLoggerConfigList(): array
```

Comment:
Configuration should come from a special delegated class - ProcessConfig => AbstractBundleConfig